text/2027-object_safe_for_dispatch.md

```diff
@@ -83,7 +83,7 @@ dispatch.
 [how-we-teach-this]: #how-we-teach-this
 
 This is just a slight tweak to how object safety is implemented. We will need
-to make sure the the official documentation is accurate to the rules,
+to make sure the official documentation is accurate to the rules,
 especially the reference.
```

Comment:
This instance of `the the` could potentially have meant to say `that the`, but I'm not sure. Both seem possible grammatically.